src/platformAccessory.ts

```diff
@@ -253,12 +253,17 @@ export class CowayPlatformAccessory {
           } else {
             fan = Fan.Low;
           }
-          return this.controlDevice([
+          await this.controlDevice([
             {
               funcId: FunctionId.Fan,
               cmdVal: fan,
             },
           ]);
+          // setting fan manually sets to manual mode
+          airPurifierService.updateCharacteristic(
```

Comment:
This should actually read automatically at status update but no harm in setting it here i'd imagine. It wasn't throwing any errors for me though.